prow/config.gen.yaml

```diff
@@ -785,6 +785,41 @@ presubmits:
             memory: 4Gi
 
   maistra/proxy:
+  - name: proxy-unit-clang
+    trigger: (?m)^/test( | .* )unit,?($|\s.*)
+    decorate: true
+    decoration_config:
+      timeout: 6h0m0s
+    always_run: true
```

Comment:
why run this always if it's experimental? you can also have jobs that have to be triggered explicitly by calling their `/command`